src/editor/ModelViewer.cpp

```diff
@@ -402,7 +402,7 @@ void ModelViewer::OnModelLoaded()
 	ResetThrusters();
 
 	SceneGraph::DumpVisitor d(model);
-	model->GetRoot()->Accept(d);
```

Comment:
Is there a reason this is in this review?

Reply:
Oops